### ISSUE TITLE
Wait For Image - ignore @notWantedImage

### ISSUE THREAD

**Issue by SudPiva** [rainmanwy/robotframework-SikuliLibrary#114]
Using the keyword "Wait For Image" doesn't achieve what is it supposed to do. It ignores the image defined in @notWantedImage

made the same Keyword using exist and that works - doesnt have timeout :-(
`
Image Not Visible

    [Arguments]  ${yesImage}  ${noImage}  ${timeout}
    Sleep    ${timeout}
    ${isExist} =       Exists    ${yesImage}
    ${isNotExist} =    Exists    ${noImage}
    Should Be True               ${isExist}
    Should Not Be True           ${isNotExist}
`

Using robotframework-SikuliLibrary 1.0.8

**Comment by rainmanwy** [comment]
@SudPiva , so in your environment, "wantedImage" and "notWantedImage" may exist at the same time, right?

I think "wantedImage" and "notWantedImage" are mutex each other, so when keyword find "wantedImage" and then return.

**Comment by SudPiva** [comment]
> @SudPiva , so in your environment, "wantedImage" and "notWantedImage" may exist at the same time, right?
> 
> I think "wantedImage" and "notWantedImage" are mutex each other, so when keyword find "wantedImage" and then return.

yes thay may exist at the same time. I was not sure from the description of that keyword if it supposed to work like that or not. So I guess to close this, if this is the wanted behavior.

**Comment by jpeltonen** [comment]
I think it should raise exception if there are "wantedImage" and "notWantedImage" found at the same time.
